src/_gettsim/groupings.py

```diff
@@ -230,3 +234,32 @@ def wthh_id_numpy(
             result.append(current_hh_id * 100)
 
     return numpy.asarray(result)
+
+
+def _fail_if_more_than_one_fg_in_hh(
+    hh_id: numpy.ndarray[int],
+    fg_id: numpy.ndarray[int],
+):
+    """
+    Fail if there is more than one `fg_id` in a household.
+
+    Parameters
+    ----------
+    hh_id : numpy.ndarray[int]
+        Array of household IDs.
+    fg_id : numpy.ndarray[int]
+        Array of family group IDs.
+    """
+    unique_hh_ids = numpy.unique(hh_id)
+    error_msg = (
+        "There is at least one household with more than one `fg_id`. GETTSIM does "
+        "not support the endogenous creation of Bedarfsgemeinschaften in this case "
+        "yet. Please provide `bg_id` and `wthh_id` yourself."
+    )
+
+    for idx in unique_hh_ids:
```

Comment:
```suggestion
    for this_hh_id in unique_hh_ids:
```
Makes it clearer what the loop is over.